apps/hermes/server/src/api/rest/v2/latest_price_updates.rs

```diff
@@ -57,6 +57,10 @@ pub struct LatestPriceUpdatesQueryParams {
     /// If true, include the parsed price update in the `parsed` field of each returned feed. Default is `true`.
     #[serde(default = "default_true")]
     parsed: bool,
+
+    /// If true, invalid price IDs in the `ids` parameter are ignored. Default is `false`.
+    #[serde(default)]
+    ignore_invalid_price_ids: bool,
```

Comment:
I suggest somehow specifying that it only affects V2 routes, either by name or comment.

Reply:
Good idea